### ISSUE TITLE
Canvas+ not working for HCC Canvas

### ISSUE THREAD

**Issue by desidient** [canvasplus/extension#72]
I tried changing the sidebar display for Canvas but when I click "sidebar" in canvas+ it doesn't do anything

**Comment by jadakiss8** [comment]
yeah i just added it to chrome and it's not changing anything. i added Better Canvas extension and it works fine so maybe it's an admin block?
